apps/mull-ui/src/app/components/chat-bubble/chat-bubble.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './chat-bubble.scss';
+
+export interface chatBubbleProps {
+  isHost?: boolean;
```

Comment:
```suggestion
  isCurrentUser?: boolean;
```
Just a small suggestion about the prop name. Technically, everyone commenting on the Announcements page is a host/co-host, so the name of this boolean is not too representative. The chat colour should be green if it the comment is the current-logged-in-user, and them being a host or not isn't really a factor in that.